pbft/pbft.cpp

```diff
@@ -1307,7 +1307,7 @@ pbft::is_valid_newview_message(const pbft_msg& theirs, const bzn_envelope& origi
         viewchange_envelopes_from_senders[viewchange_env.sender()] = viewchange_env;
     }
 
-    pbft_msg ours = this->build_newview(theirs.view(), viewchange_envelopes_from_senders);
+    pbft_msg ours = this->build_newview(theirs.view(), viewchange_envelopes_from_senders, false);
     if (ours.pre_prepare_messages_size() != theirs.pre_prepare_messages_size())
```

Comment:
Cool, while checking for new view message validity we do not need to worry about adding requests we don't have.

Reply:
No the default should be true. Changing it.